### PR TITLE
fix(lap-379): update verify flow when user enter invalid otp

### PR DESCRIPTION
**Pull request by Huy203**
Update back to previous otp input when user delete input of current otp input

### CODE REVIEW

File: locales/vi/auth.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
     "instruction": "Mã xác minh đã được gửi qua email đến {{email}}",
     "checkOtpButton": "Kiểm tra OTP",
     "noOtp": "Chưa nhận được mã OTP?",
-    "resendCode": "Gửi lại mã {{time}} giây"
+    "resendCodeWithTime": "Gửi lại mã {{time}} giây",
```
Comment (vmathu) [inline locales/vi/auth.json:42]:
```suggestion
    "resendCodeWithTime": "Gửi lại mã sau {{time}} giây",
```

File: app/auth/(forgot-password)/verify.tsx [hunk 2]
```diff
@@ -94,7 +94,11 @@ const Verify = () => {
                         const newCode = [...field.value];
                         newCode[i] = text;
                         field.onChange(newCode);
-                        if (text.length === 1) {
+                        if (text.length === 0) {
```
Comment (NganTrucLe) [inline app/auth/(forgot-password)/verify.tsx:97]: define a function outside, no complex inline logic

File: components/molecules/NavigationBar.tsx [hunk 3]
```diff
@@ -24,31 +27,44 @@ export const NavigationBar = ({
...
-                  }
-                }}>
-                <Icon color='black' />
-              </Pressable>
-            ))}
+        <View
+          className={cn(
+            'flex h-11 flex-row items-center',
+            (displayStyle === 'center' && 'justify-center') ?? 'justify-between'
+          )}>
```
Comment (NganTrucLe) [inline components/molecules/NavigationBar.tsx:40]: change to StyleSheet

File: components/molecules/NavigationBar.tsx [hunk 4]
```diff
@@ -24,31 +27,44 @@ export const NavigationBar = ({
...
+                      } else {
+                        router.dismiss();
+                      }
+                    }}>
+                    <Icon color='black' />
+                  </Pressable>
+                ))}
+            </>
+          )}
           {headerTitle && <Text className='font-ibold text-title-4 text-black'>{headerTitle}</Text>}
```
Comment (NganTrucLe) [inline components/molecules/NavigationBar.tsx:63]: change to StyleSheet

**Review by NganTrucLe — CHANGES_REQUESTED** [review]

**Review by NganTrucLe — APPROVED** [review]

**Pull request merged**